### ISSUE TITLE
Lighting seems unusual when on the dirt in warehouse

### ISSUE THREAD

**Issue by Peter0x44** [nzp-team/nzportable#1048]
**Describe the bug**
The zombies turn black when stepping on the dirt in the corner of the first room of warehouse.
The same happens with the player's viewmodel, though it's not as obvious

**To Reproduce**
Start a game on warehouse
Wait for zombies to spawn
Stand on dirt
Wait

**Expected behavior**
More normal looking lighting

**Screenshots/Footage**

https://github.com/user-attachments/assets/9898f8ad-b78c-47e0-9584-2bfe236b2cd1

**Affected Platforms**
I tested FTEQW and PSP through ppsspp

**Comment by Peter0x44** [comment]
Supercedes #1041

**Comment by MotoLegacy** [comment]
Please elaborate, are you saying this also occurs on PSP, equally as intense as well? or?

**Comment by Peter0x44** [comment]
Yes. There's also some boxes that look black elsewhere in the map, iirc.

**Comment by MotoLegacy** [comment]
Please provide that demonstration when you can, thanks

**Comment by Peter0x44** [comment]
This issue is more obvious in B1ooD Version 3

![Image](https://github.com/user-attachments/assets/dd1daa92-aee8-46e3-a36d-0bdad2880263)
